flexbox/src/main/java/com/google/android/flexbox/FlexboxLayout.java

```diff
@@ -322,7 +322,7 @@ public void addView(View child, int index, ViewGroup.LayoutParams params) {
      * @return an array which have the reordered indices
      */
     private int[] createReorderedIndices(View viewBeforeAdded, int indexForViewBeforeAdded,
-            ViewGroup.LayoutParams paramsForViewBeforeAdded) {
+                                         ViewGroup.LayoutParams paramsForViewBeforeAdded) {
```

Comment:
Could you revert the unneeded format changes?

Ditto for the rest of the format changes below.